core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -45,24 +46,20 @@
     private static final double MAX_ENTRIES_FACTOR = 2;
     private static final double MAX_IDLE_ENTRIES_FACTOR = 5;
 
-    // when chunk is frozen, all of the elements in pending puts array will be this OpData
-    private static final OpData FROZEN_OP_DATA =
```

Comment:
Please do not remove this... I still would like to live with the possibility that rebalance will return to be non-blocking one day...

Reply:
I don't think we should leave unused code in the codebase. If we need it back, that's what version control is for.